src/RealTime/Localization/Translations/ko.xml

```diff
@@ -129,99 +129,99 @@
-    <translation id="MILESTONE_COMBINED_PASSED_TIME_DESC" value="{1} × {unit} 에 대한 {0}을 통과"/>
-    <translation id="MILESTONE_COMBINED_TIME_DESC" value="{0} × {unit} 를 유지"/>
-    <translation id="PARKINFOPANEL_CURRENTVISITORS" value="저번 {unit} 중 방문객 : {0}"/>
+    <translation id="MILESTONE_COMBINED_PASSED_TIME_DESC" value="{0}을 통과한 지 {1} × {unit}"/>
+    <translation id="MILESTONE_COMBINED_TIME_DESC" value="{0} × {unit}를 유지"/>
+    <translation id="PARKINFOPANEL_CURRENTVISITORS" value="저번 {unit} 동안 방문객 : {0}"/>
     <translation id="PARKINFOPANEL_INCOME" value="수익: {0} / {unit}"/>
-    <translation id="PROFIT_TT_TOTAL_PUBTRANS" value="저번 {unit} 중 총 대중교통으로 인한 수익"/>
-    <translation id="STADIUMINFO7" value="저번 12 × {unit} 당 경기당 수익 :"/>
+    <translation id="PROFIT_TT_TOTAL_PUBTRANS" value="저번 {unit} 동안 총 대중교통으로 벌어들인 수익"/>
+    <translation id="STADIUMINFO7" value="저번 12 × {unit} 당 경기당 수익:"/>
     <translation id="STADIUMINFO9" value="저번 {unit} 중 수익:"/>
-    <translation id="TOOL_WATER_ESTIMATE" value="총 가용량 : {0:N0} m³/{unit}"/>
-    <translation id="TRANSPORT_LINE_PASSENGERS_WALKINGTOUR" value="저번 {unit} 당 방문객 : {0} 시민들, {1} 관광객들"/>
-    <translation id="TRANSPORT_LINE_PASSENGERS" value="저번 {unit} 당 승객 : {0} 시민들, {1} 관광객들"/>
-    <translation id="AIINFO_INDUSTRY_PRODUCTION_RATE" value="Production rate: {0:N0} units per {unit}"/>
-    <translation id="AIINFO_POST_SORTING_RATE" value="Sorting rate: {0:N0} pieces of mail per {unit}"/>
-    <translation id="AIINFO_TOLL_BOOTH_VEHICLE_COUNT" value="Cars charged last {unit}: {0:N0}"/>
-    <translation id="ECONOMYPANEL_TOOLTIP_INDUSTRIAL_PLAYERINDUSTRY" value="Income from zoned industry and industry areas last {unit}"/>
-    <translation id="EXPENSES_TT_PLAYERINDUSTRY" value="Industry Area Expenses per {unit}"/>
-    <translation id="INCOME_TT_ROADTOLLS" value="Road Toll Income per {unit}"/>
-    <translation id="INCOME_TT_TOTAL_ROADTOLLS" value="Total Income from Road Tolls last {unit}"/>
-    <translation id="INDUSTRYPANEL_ACQUISITION_TOOLTIP" value="Acquisition per {unit}"/>
-    <translation id="INDUSTRYPANEL_ACQUISITIONCOST_TOOLTIP" value="Cost of Importing the Resource per {unit}"/>
-    <translation id="INDUSTRYPANEL_OUTPUT_TOOLTIP" value="Output per {unit}"/>
-    <translation id="INDUSTRYPANEL_OUTPUTINCOME_TOOLTIP" value="Income from Selling the Resource per {unit}"/>
-    <translation id="INDUSTRYPANEL_PROFIT_TOOLTIP" value="Profit per {unit}"/>
+    <translation id="TOOL_WATER_ESTIMATE" value="총 가용량 : {0:N0} m³ /{unit}"/>
+    <translation id="TRANSPORT_LINE_PASSENGERS_WALKINGTOUR" value="저번 {unit} 당 방문객: {0} 시민들, {1} 관광객들"/>
+    <translation id="TRANSPORT_LINE_PASSENGERS" value="저번 {unit} 당 승객: {0} 시민들, {1} 관광객들"/>
+    <translation id="AIINFO_INDUSTRY_PRODUCTION_RATE" value="생산률: {unit} 당 {0:N0} 유닛"/>
+    <translation id="AIINFO_POST_SORTING_RATE" value="편지 분류률: {unit} 당 {0:N0} 개"/>
+    <translation id="AIINFO_TOLL_BOOTH_VEHICLE_COUNT" value="저번 {unit}당 {0:N0}의 통행 수익"/>
```

Comment:
In some translations you have spaces around `{unit}`, while in others not. Maybe make all them consistent?